Voice/ContactFlows/README.md

```diff
@@ -8,3 +8,5 @@ Use these contact flows to perform common functions when using Service Cloud Voi
 * [Sample_SCV_Inbound_Flow_With_Transcription](Sample_SCV_Inbound_Flow_With_Transcription): Use this contact flow to define the customer experience for an inbound call where transcription is enabled.
 * [Sample_SCV_Outbound_Flow_With_Transcription](Sample_SCV_Outbound_Flow_With_Transcription): Use this contact flow to define the customer experience for an outbound call (that is, the agent calls the customer) where transcription is enabled.
 * [Sample_SCV_Queue_Transfer](Sample_SCV_Queue_Transfer): Use this contact flow to define the customer experience when a call is transferred to a queue.
+* [Sample_REST_Check_For_Open_Cases](Sample_REST_Check_For_Open_Cases): Use this contact flow to demonstrate the `InvokeSalesforceRestApiFunction` Lambda function by checking for open cases.
```

Comment:
All the contact flows has Sample_SCV naming conventions. Should we follow for these new one ?

Reply:
Good feedback. Updated accordingly.